PowerPointLabs/PowerPointLabs/ELearningLab/Service/AudioSettingService.cs

```diff
@@ -20,12 +25,23 @@ public static ObservableCollection<IVoice> preferredVoices
 
         public static void ShowSettingsDialog()
         {
-            AudioSettingsDialogWindow dialog = AudioSettingsDialogWindow.GetInstance();
-            AudioMainSettingsPage.GetInstance().SetAudioMainSettings(
+            CustomTaskPane eLearningTaskpane = ActionFrameworkExtensions.GetTaskPane(typeof(ELearningLabTaskpane));
```

Comment:
This line is causing the pragma 618.

Reply:
Yes because I want default voice labels on e-learning items to be updated as well when default voice is changed through settings. Obtaining `eLearningLabTaskpane` instance from global settings and attach a delegate to ` voice setting changed event` seems to be the only way to achieve this.